exec/vector/src/main/codegen/templates/VariableLengthVectors.java

```diff
@@ -253,7 +253,7 @@ public int getPayloadByteCount() {
       // If 1 or more values, then the last value is set to
       // the offset of the next value, which is the same as
       // the length of existing values.
-      return a.get(count-1);
+      return a.get(count-1) + offsetVector.getPayloadByteCount();
```

Comment:
Don't understand this new addition; doesn't the get(count-1) already reports all the memory used by this varchar, including the overhead ?  Maybe need to explain in the comment.